### PR TITLE
Use tahoe-sites.api - Patch 02

### PR DESCRIPTION
**Pull request by shadinaif**
## Change description

Use tahoe-sites.api - Patch 02:
* Use tahoe-sites instead of `lms.djangoapps.instructor.sites`
* Remove UserOrganizationMapping from CMS

## Type of change
- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

https://appsembler.atlassian.net/browse/RED-2709

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Comment by OmarIthawi** [comment]
same here @shadinaif, please check the coverage report for missing tests:

 - https://coveralls.io/builds/47642513/source?filename=common%2Fdjangoapps%2Fstudent%2Fmodels.py#L1632

File: requirements/edx/appsembler.txt [hunk 1]
```diff
@@ -22,7 +22,8 @@ https://github.com/edx-solutions/xblock-google-drive/archive/589d9f51f9b.tar.gz
 
 # Tahoe plugins and customizations
 django-tiers==0.2.4
-tahoe-sites==0.1.5
+# tahoe-sites==0.1.5 plus new APIs
+-e git+https://github.com/appsembler/tahoe-sites.git@980c7c70125259a7c8041f82ea86042c565024bb#egg=tahoe-sites==0.1.5-newapis
```
Comment (OmarIthawi) [inline requirements/edx/appsembler.txt:26]: right?

```suggestion
tahoe-sites==0.1.6
```

**Review by OmarIthawi — APPROVED** [review]
looks good, thanks shadi!

**Review by thraxil — APPROVED** [review]

File: requirements/edx/appsembler.txt (hunk 1 above)
Comment (shadinaif) [reply requirements/edx/appsembler.txt]: sure, thank you!

**Comment by thraxil** [comment]
Anything holding this up?

**Pull request merged**